packages/now-next/src/index.ts

```diff
@@ -376,8 +376,18 @@ export const build = async ({
     await writeNpmRc(entryPath, process.env.NPM_AUTH_TOKEN);
   }
 
-  console.log('Installing dependencies...');
-  await runNpmInstall(entryPath, [], spawnOpts, meta);
+  const installTime = Date.now();
+  if (typeof installCommand === 'string') {
+    console.log(`Running "install" command: \`${installCommand}\`...`);
+    await execCommand(installCommand, {
+      ...spawnOpts,
+      cwd: entryPath,
+    });
```

Comment:
Let's also pass `prettyCommand: installCommand` here so errors look nice.

You'll probably need to change the type of the execCommand() function to `SpawnOptionsExtended`

Reply:
Seems that [`execCommand()` already passes `prettyCommand`](https://github.com/vercel/vercel/blob/e2db7c77347fe7698a76df8f31062b9c3b1490b4/packages/now-build-utils/src/fs/run-user-scripts.ts#L110).